src/SprykerEco/Zed/Econda/Dependency/Facade/EcondaToPriceBridge.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Dependency\Facade;
+
+class EcondaToPriceBridge implements EcondaToPriceBridgeInterface
+{
+    /** @var \Spryker\Zed\Price\Business\PriceFacadeInterface */
+    protected $facade;
+
+    /**
+     * @param \Spryker\Zed\Price\Business\PriceFacadeInterface $facade
+     */
+    public function __construct($facade)
+    {
+        $this->facade = $facade;
+    }
+
+    /**
+     * @param string $sku
+     * @param string|null $priceTypeName
+     *
+     * @return int
```

Comment:
Is it int or float? You have both declared in different places.. PHPStan can find this as well in level 5 by the way.

Reply:
EcondaToPriceBridge::getPriceBySku() returns int cos that the return type of the PriceFacade. Then, EcondaProductCollector::getPriceBySku()  returns float cos its dividing the value it gets from EcondaToPriceBridge by 100

Reply:
Fixed